docs/assets/kafka-source.svg

```diff
@@ -0,0 +1,113 @@
+                </g>
+                <g id="Source-1" transform="translate(82.000000, 56.000000)">
+                    <circle id="Oval-Copy-2" fill="#1BE6FF" cx="37.5" cy="37.5" r="37.5"></circle>
+                    <path d="M28.965,27.982 L28.965,31.44 L23.531,31.44 L23.531,46 L19.475,46 L19.475,31.44 L14.041,31.44 L14.041,27.982 L28.965,27.982 Z M39.183,46.364 C36.253652,46.364 34.0220077,45.5493415 32.488,43.92 C30.9539923,42.2906585 30.187,39.9766817 30.187,36.978 C30.187,35.4873259 30.4209977,34.1570058 30.889,32.987 C31.3570023,31.8169941 31.9983293,30.829004 32.813,30.023 C33.6276707,29.216996 34.5983277,28.6060021 35.725,28.19 C36.8516723,27.7739979 38.0736601,27.566 39.391,27.566 C40.1536705,27.566 40.8469969,27.6223328 41.471,27.735 C42.0950031,27.8476672 42.6409977,27.9776659 43.109,28.125 C43.5770023,28.2723341 43.9669984,28.4239992 44.279,28.58 C44.5910016,28.7360008 44.8163326,28.8573329 44.955,28.944 L43.785,32.22 C43.2303306,31.9253319 42.5846703,31.674001 41.848,31.466 C41.1113297,31.257999 40.2750047,31.154 39.339,31.154 C38.7149969,31.154 38.104003,31.257999 37.506,31.466 C36.907997,31.674001 36.3793356,32.0076644 35.92,32.467 C35.4606644,32.9263356 35.0923347,33.5199964 34.815,34.248 C34.5376653,34.9760036 34.399,35.8599948 34.399,36.9 C34.399,37.7320042 34.4899991,38.5076631 34.672,39.227 C34.8540009,39.9463369 35.1486646,40.5659974 35.556,41.086 C35.9633354,41.6060026 36.49633,42.0176651 37.155,42.321 C37.81367,42.6243348 38.6109953,42.776 39.547,42.776 C40.1363363,42.776 40.6649977,42.7413337 41.133,42.672 C41.6010023,42.6026663 42.0169982,42.5203338 42.381,42.425 C42.7450018,42.3296662 43.0656653,42.2213339 43.343,42.1 C43.6203347,41.9786661 43.8716655,41.8660005 44.097,41.762 L45.215,45.012 C44.6429971,45.3586684 43.8370052,45.6706653 42.797,45.948 C41.7569948,46.2253347 40.5523402,46.364 39.183,46.364 Z M53.223,27.774 C55.9096801,27.774 57.9723261,28.2463286 59.411,29.191 C60.8496739,30.1356714 61.569,31.6826559 61.569,33.832 C61.569,35.9986775 60.8410073,37.5629952 59.385,38.525 C57.9289927,39.4870048 55.8490135,39.968 53.145,39.968 L51.871,39.968 L51.871,46 L47.815,46 L47.815,28.242 C48.6990044,28.0686658 49.6349951,27.9473337 50.623,27.878 C51.6110049,27.8086663 52.4776629,27.774 53.223,27.774 Z M53.483,31.232 C53.1883319,31.232 52.8980014,31.2406666 52.612,31.258 C52.3259986,31.2753334 52.079001,31.2926666 51.871,31.31 L51.871,36.51 L53.145,36.51 C54.549007,36.51 55.6063298,36.3193352 56.317,35.938 C57.0276702,35.5566648 57.383,34.8460052 57.383,33.806 C57.383,33.3033308 57.2920009,32.887335 57.11,32.558 C56.9279991,32.228665 56.6680017,31.9643343 56.33,31.765 C55.9919983,31.5656657 55.5803358,31.4270004 55.095,31.349 C54.6096642,31.2709996 54.0723363,31.232 53.483,31.232 Z" id="TCP" fill="#FFFFFF" fill-rule="nonzero"></path>
```

Comment:
The SVG image still refers to the `TCP` sink. Probably that was the starting point for the Kafka sink. It's just a minor nitpick but we should remember to fix this before merging.

Reply:
I've tried to modify it, but was not sure which font to use (it turned out to be Ubuntu bold), then forgot to mention it in the PR. The image is updated now.